ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
Hum, you're right I've just tested it and tinyMCE doesn't need it.

BUT, we will "potentially" need it elsewhere :
All new content/document, will have its summary wrapped in a `<p>...</p>` so we will have our templates taking this into account (as you've mentioned it for `content_card.html`)

If we don't wrap already existing summaries in a `<p>...</p>`, we will have sometime a summary wrapped (for new content) and sometime not (for old content). This is a inconsistency in the way we structure our layout.

For now, it seems not important as the presence of the `p` doesn't change a lot of thing. But it may in the future (styling the `p` element in a css). At this moment it will be difficult to add the `<p>...</p>` as it will be difficult to make the distinction between new and old contents.